models/webhook.go

```diff
@@ -110,3 +91,36 @@ func (f WebhookEventFilters) MergeWithDefaults() WebhookEventFilters {
 	}
 	return defaultFilters
 }
+
+type WebhookRegister struct {
+	OrganizationId    string
+	PartnerId         null.String
+	EventTypes        []string
+	Secret            string
+	Url               string
+	HttpTimeout       *int
+	RateLimit         *int
+	RateLimitDuration *int
+}
+
+func (input WebhookRegister) Validate() error {
+	for _, eventType := range input.EventTypes {
+		if _, err := ParseWebhookEventType(eventType); err != nil {
```

Comment:
unless ParseWebhookEventType is used somewhere else (I don't think so?), you can just replace this with something like 
```go
if !slices.Contains(validWebhookEventTypes, WebhookEventType(eventType)) {...
```